### PR TITLE
Dummy data

### PR DESCRIPTION
**Pull request by jxhncxmpbxll**
Adds seed script

### CODE REVIEW

File: database/seed.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+    for (let i = 0; i < 100; i++) {
+
+      const fakeHome = {
+        property_id: '',
+        streetAddress: faker.address.streetAddress(),
+        streetName: faker.address.streetName(),
+        state: 'The North',
+        zipcode: faker.address.zipCode(),
+        neighborhood: neighborhoods[Math.random() * (neighborhoods.length - 1)],
```
Comment (davidhk21) [inline database/seed.js]: Might want to add Math.floor to randomly choosing neighborhood?

File: database/seed.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  const dummyData = [];
+
+  const neighborhoods = ['Last Hearth', 'Crofter\'s Village', 'Hornwood', 'Wolfswood', 'Cerwyn'];
+
+  const cities = ['Ramsgate', 'Dreadfort', 'Winterfell', 'White Harbor', 'Barrowton'];
+
+    for (let i = 0; i < 100; i++) {
+
+      const fakeHome = {
+        property_id: '',
```
Comment (davidhk21) [inline database/seed.js]: what's this property_id property for?

File: database/seed.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        state: 'The North',
+        zipcode: faker.address.zipCode(),
+        neighborhood: neighborhoods[Math.random() * (neighborhoods.length - 1)],
+        city: cities[Math.floor(Math.random() * (cities.length - 1))],
+        bedrooms: Math.floor(Math.random() * 3),
+        bathrooms: Math.floor(Math.random() * 5),
+        sqft: (Math.random() * (5,100 - 600) + 600),
+        publishDate: faker.date.past(1),
+        price: Math.random() * (2740000 - 150000) + 150000,
+        img_url: ''
```
Comment (davidhk21) [inline database/seed.js]: I'm assuming that you'll be adding these image urls later?

File: server/index.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
+const express = require('express');
+const app = express();
+const bodyParser = require('body-parser');
+const db = require('../database/model.js');
+const seed = require(('../database/seed.js');)
+const helpers = require('../helpers/realtor.js');
+
+const PORT = 3000;
+
+app.use(bodyParser.json());
```
Comment (davidhk21) [inline server/index.js]: just for future reference, but you can actually just do "app.use(express.json());". no need to install and require body parser 👍

File: webpack.config.js [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+module.exports = {
+  entry: `${SRC_DIR}/app.jsx`,
+  output: {
+    filename: 'bundle.js',
+    path: DIST_DIR
+  },
+  module: {
+    rules: [
+      {
+        test: /\.jsx?/,
```
Comment (davidhk21) [inline webpack.config.js]: If this doesn't work, I'd try "/\.jsx?$/". But it's cool if it does work!

**Review by davidhk21 — APPROVED** [review]
thought your code looked clean! great job :)

File: database/seed.js (hunk 1 above)
Comment (jxhncxmpbxll) [reply database/seed.js]: Great catch! I went back and fixed it

File: database/seed.js (hunk 2 above)
Comment (jxhncxmpbxll) [reply database/seed.js]: It was for an api call that I'm no longer using. I went ahead and removed it

File: database/seed.js (hunk 3 above)
Comment (jxhncxmpbxll) [reply database/seed.js]: They were added shortly after your review!

File: server/index.js (hunk 4 above)
Comment (jxhncxmpbxll) [reply server/index.js]: That's really good to know! Thank you!

File: webpack.config.js (hunk 5 above)
Comment (jxhncxmpbxll) [reply webpack.config.js]: Seems to be working!

**Pull request merged**